ivy_tests/test_ivy/helpers/testing_helpers.py

```diff
@@ -428,6 +428,10 @@ def handle_frontend_test(
     generate_frontend_arrays
         A search strategy that generates a list of boolean flags for array inputs to
         be frontend array
+
```

Comment:
Hello @Chinemelu4, can you please revert those changes that aren't related to the issue? Thanks! 😊